backend_new/src/services/application.service.ts

```diff
@@ -373,6 +375,19 @@ export class ApplicationService {
       );
     }
 
+    // Calculate geocoding preferences after save and email sent
+    if (listing.jurisdictions?.enableGeocodingPreferences) {
+      try {
+        void this.geocodingService.validateGeocodingPreferences(
+          mapTo(Application, rawApplication),
+          mapTo(Listing, listing),
+        );
+      } catch (e) {
+        // If the geocoding fails it should not prevent
```

Comment:
Just a nit but not 100p clear what should not be prevented here

Reply:
Looks like I started writing a sentence and then failed to finish it. I'll update this